jenkins_rock/rockcraft.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: jenkins
+summary: Jenkins rock
+description: Jenkins OCI image for the Jenkins charm
+version: "1.0"
+base: ubuntu:22.04
+build-base: ubuntu:22.04
+license: Apache-2.0
+platforms:
+  amd64:
+services:
```

Comment:
As I understand it, the `services` are overridden by the pebble plan defined in the charm, meaning we can drop the key

Reply:
That's correct, however, I've decided to add the services definition here for easier docker spin up w/ pebble to facilitate rockcraft build and load :) Would that be okay with you?
I do understand the drawbacks of having multiple definitions of pebble services though. 
As far as I can tell from the [list of `rockcraft.yaml`s](https://docs.google.com/spreadsheets/d/1i1G230Ch2r2ZpUT-yrYLEOzqwQd2UZN-FbcmfueypqU/edit#gid=0) we have inside the org, there's no standard on whether to add the service definitions or not.
One without: [OIDC](https://github.com/canonical/oidc-authservice-rock/blob/main/rockcraft.yaml)
One with: [kubeflow](https://github.com/canonical/kubeflow-rocks/blob/main/centraldashboard/rockcraft.yaml).

Reply:
No, there's no consensus, at least for now. Pebble will be always injected as entrypoint, but I'm wondering if, when defining the service in the rockcraft.yaml, it would make more sense to `merge` the new layer with this one during the pebble setup within the charm https://juju.is/docs/sdk/interact-with-pebble